src/Streamly/Internal/Data/Array/Stream/Mut/Foreign.hs

```diff
@@ -99,7 +99,7 @@ packArraysChunksOf n (D.Stream step state) =
     step' gst (SpliceInitial st) = do
         when (n <= 0) $
             -- XXX we can pass the module string from the higher level API
-            error $ "Streamly.Internal.Data.Array.Storable.Foreign.Mut.Types.packArraysChunksOf: the size of "
+            error $ "Streamly.Internal.Data.Array.Foreign.Mut.Types.packArraysChunksOf: the size of "
```

Comment:
This should be `Streamly.Internal.Data.Array.Stream.Mut.Foreign.packArraysChunksOf`